### PR TITLE
[SDN-1317] Add Data collection for ovn-ipsec encrypted clusters 

### PR DESCRIPTION
**Pull request by astoycos**
This PR adds some data collection that is specific to ovn-ipsec encrypted clusters

### CODE REVIEW

**Comment by astoycos** [comment]
/hold this shouldn't be merged until we can test on a fullfledged OCP cluster, we need `ovn-ipsec` to finish landing first 

Currently tested on a local kind cluster

**Comment by astoycos** [comment]
@markdgray

File: collection-scripts/gather_network_logs [hunk 1]
```diff
@@ -14,6 +14,36 @@ function gather_multus_data {
...
+    PIDS+=($!)
+
+    oc -n openshift-ovn-kubernetes exec "${IPSEC_POD}" -- bash -c "certutil -L -d sql:/etc/ipsec.d" \
+    > "${NETWORK_LOG_PATH}"/"${IPSEC_POD}"_active_libreswan_certs & 
+    PIDS+=($!)
+
+    oc -n openshift-ovn-kubernetes exec "${IPSEC_POD}" -- bash -c "ovs-vsctl list Open_vSwitch" \
+    > "${NETWORK_LOG_PATH}"/"${IPSEC_POD}"_ & 
+    PIDS+=($!)
+
```
Comment (markdgray) [inline collection-scripts/gather_network_logs:35]: Can you also check if ovn has ipsec enabled? I am not sure if I mentioned this to you. You can either print the entire table

`ovn-nbctl list nb_global`

or print the actual setting

`ovn-nbctl get nb_global . ipsec`

This needs to be run on ovnkube-master in the nbdb container.

File: collection-scripts/gather_network_logs [hunk 2]
```diff
@@ -14,6 +14,36 @@ function gather_multus_data {
   oc describe overlappingrangeipreservations.whereabouts.cni.cncf.io -A > "${NETWORK_LOG_PATH}"/overlappingrangeipreservations  2>&1 & PIDS=($!)
 }
 
+function gather_ovn_ipsec_data { 
+  echo "INFO: Gathering ovn-ipsec data"
+  for IPSEC_POD in ${IPSEC_PODS[@]}; do
```
Comment (markdgray) [inline collection-scripts/gather_network_logs:19]: Where do you populate `${IPSEC_PODS[@]}?

File: collection-scripts/gather_network_logs (hunk 1 above)
Comment (astoycos) [reply collection-scripts/gather_network_logs:39]: Done

Comment (markdgray) [reply collection-scripts/gather_network_logs:39]: This command will probably need to run on ovnkube-master pod in the nbdb container?

**Pull request merged**